spacelift/data_stack.go

```diff
@@ -321,6 +321,11 @@ func dataStack() *schema.Resource {
 				Description: "ID (slug) of the stack",
 				Required:    true,
 			},
+			"terraform_smart_sanitization": {
+				Type:        schema.TypeBool,
+				Description: "Indicates whether runs on this will use terraform's sensitive value system to sanitize the outputs of Terraform state and plans in spacelift.",
```

Comment:
I think it would be good to add to the description: "instead of sanitizing everything". Cause reading this, I'd expect that if this is false, then there is no sanitization at all.

Reply:
That's a fair point. I'll edit the description now to clarify this point.